### PR TITLE
Do not remove downloads/.gitignore

### PR DESCRIPTION
**Pull request by mhdirkse**

### CODE REVIEW

File: cypress/downloads/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+*
```
Comment (celmoussaoui) [inline cypress/downloads/.gitignore:1]: Isnt it supposed to be `.*` instead of `*`

Comment (mhdirkse) [reply cypress/downloads/.gitignore:1]: I tested it by putting some arbitrary file in the downloads directory. After updating `.gitignore` as done here, the file was no longer shown with `git status`. Do you want me to change to `.*` and test that?

Comment (celmoussaoui) [reply cypress/downloads/.gitignore:1]: Yes, can you test it?

Comment (mhdirkse) [reply cypress/downloads/.gitignore:1]: `.*` does not do it. Only `*`.

Comment (celmoussaoui) [reply cypress/downloads/.gitignore:1]: Okay

**Pull request merged**